ui/mirage/factories/sync-association.js

```diff
@@ -6,7 +6,7 @@
 import { Factory } from 'ember-cli-mirage';
 
 export default Factory.extend({
-  accessor: 'kv_eb4acbae',
```

Comment:
My understanding was the backend was going to add `mount`, while keeping `accessor` - if that's the case should we leave this key for API parity?

Reply:
I don't think it's necessary if we aren't consuming it in any way but it wouldn't hurt either. I can add it back with a comment.